### ISSUE TITLE
image size and image view aspect ratio

### ISSUE THREAD

**Issue by kswang1029** [idia-astro/carta-python#86]
When we use `Session.set_view_area()` to configure a view area in a headless mode, it refers to the image size from  `session.save_rendered_view('test.png')` (ie the red box in the attached figure). In practice, we better consider the view_area as the "field of view" of the image viewer (ie the green box in the attached figure). In this case, it provides users the control of the aspect ratio of the "image" in interest directly. Then users may use other tools to compose a grid of images for presentation purposes.

Is it possible to support this? This may need the support from carta-frontend? @veggiesaurus @YuHsuan-Hwang 

![Screen Shot 2023-03-20 at 14 31 14](https://user-images.githubusercontent.com/20819712/226264436-c1319dba-98e4-4e92-90d1-6ee666b3a71a.png)

**Comment by YuHsuan-Hwang** [comment]
@kswang1029 Yes, I think frontend now supports it. We can get the paddings from `OverlayStore.paddingLeft/Right/Top/Bottom`, and then call `set_view_area` with green box size + paddings. But we will need to resize it when overlay settings are changed (i.e. paddings changed). Also note that `set_view_area` includes multiple images.
